components/post-context/context.ts

```diff
@@ -0,0 +1,19 @@
+import { createContext, useContext } from '@wordpress/element';
+
+export const DEFAULT_POST_CONTEXT = {
```

Comment:
What if you add the `PostContextType` type to the `DEFAULT_POST_CONTEXT` const to make it more type safety? 🙂